host-bin/edit-chroot

```diff
@@ -385,8 +432,22 @@ for NAME in "$@"; do
         fi
         echo -n "  Restoring $src to $CHROOT..." 1>&2
         mkdir -p "$CHROOT"
+        if [ -f "$src.part01" ]; then
+            tarsrc="`mktemp -u --tmpdir=/tmp 'crouton-restore.XXX'`"
+            mkfifo -m 600 "$tarsrc"
+            # Pipe through dd to eat the EOFs
```

Comment:
When cat goes from file to file, it seems to generate EOFs between each file that screws up tar.  Piping through dd turns it into a standard stream.  Not really sure why this is.